docs/guides.md

```diff
@@ -2,8 +2,12 @@
 
 * [Get started with the Code42 command-line interface (CLI)](userguides/gettingstarted.md)
 * [Configure a profile](userguides/profile.md)
-* [Ingest file events or alerts into a SIEM](userguides/siemexample.md)
+* [Ingest file events, alerts, and audit logs into a SIEM](userguides/siemexample.md)
```

Comment:
This seems like it could be a little wordy - would it better as just `Ingest Data into a SIEM` ?